src/Types/TypeTelepen.php

```diff
@@ -0,0 +1,186 @@
+<?php
+/**
+ * Adapted by Darren Stephens <darren.stephens@durham.ac.uk>
+ * from Java implementation of Telepen by <rstuart114@gmail.com> Robin Stuart 
+ * at https://github.com/woo-j/OkapiBarcode which uses the 
+ * Apache License 2.0 http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Implements Telepen (also known as Telepen Alpha), and Telepen Numeric.
+ *
+ * Telepen can encode ASCII text input and includes a modulo-127 check digit.
+ * Telepen Numeric allows compression of numeric data into a Telepen symbol. Data
+ * can consist of pairs of numbers or pairs consisting of a numerical digit followed
+ * by an X character. Telepen Numeric also includes a mod-127 check digit.
+ */
+
+namespace Picqer\Barcode\Types;
+
+use Picqer\Barcode\Barcode;
+use Picqer\Barcode\BarcodeBar;
+use Picqer\Barcode\Exceptions\InvalidFormatException;
+
+define('TELEPEN_START_CHAR', '_');
+define('TELEPEN_STOP_CHAR',  'z');
+define('TELEPEN_ALPHA',   'alpha');
+define('TELEPEN_NUMERIC', 'numeric');
```

Comment:
I don't like these broad define's in de codebase. I think these can be private constants in the class itself, so it is encapsuled from the rest of the application.

Reply:
changed to `const`